packages/vertica-nodejs/test/integration/client/query-error-handling-prepared-statement-tests.js

```diff
@@ -42,84 +42,4 @@ suite.test('client end during query execution of prepared statement', function (
       client.end()
     })
   )
-})
-
-function killIdleQuery(targetQuery, cb) {
-  var client2 = new Client(helper.args)
-  var pidColName = 'procpid'
-  var queryColName = 'current_query'
-  client2.connect(
-    assert.success(function () {
-      helper.versionGTE(
-        client2,
-        90200,
-        assert.success(function (isGreater) {
-          if (isGreater) {
-            pidColName = 'pid'
-            queryColName = 'query'
-          }
-          var killIdleQuery =
-            'SELECT ' +
-            pidColName +
-            ', (SELECT pg_terminate_backend(' +
-            pidColName +
-            ')) AS killed FROM pg_stat_activity WHERE ' +
-            queryColName +
-            ' = $1'
-          client2.query(
-            killIdleQuery,
-            [targetQuery],
-            assert.calls(function (err, res) {
-              assert.ifError(err)
-              assert.equal(res.rows.length, 1)
-              client2.end(cb)
-              assert.emits(client2, 'end')
-            })
-          )
-        })
-      )
-    })
-  )
-}
-
-suite.test('query killed during query execution of prepared statement', function (done) {
```

Comment:
Why are we removing this test?

Reply:
Postgres-specific functionality to kill a specific query.